my_homework_10/main.js

```diff
@@ -0,0 +1,66 @@
+// EX1 : In một bảng nhân của một số bất kỳ với các số từ 1 tới 10
+
+for(let alter = 1; alter <= 10; alter++) {
+    const deFault = 8;
+    console.log(deFault + " x " + alter + " = " + (deFault * alter));
+}
+
+// EX2 : Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 In các số chẵn từ 1 đến n
+
+function parameters(number) {
```

Comment:
```suggestion
function parameters (number) {
```
- Đặt lại tên hàm
- Khi khai báo hàm thì thêm dấu cách trước `(`